diff --git a/node_modules/@studiometa/webpack-config/src/presets/prototyping/index.js b/node_modules/@studiometa/webpack-config/src/presets/prototyping/index.js
index 8ba2e75..3bb34e5 100644
--- a/node_modules/@studiometa/webpack-config/src/presets/prototyping/index.js
+++ b/node_modules/@studiometa/webpack-config/src/presets/prototyping/index.js
@@ -6,7 +6,7 @@ import * as glob from 'glob';
 import path from 'path';
 import merge from 'lodash.merge';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import minimatch from 'minimatch';
+import * as minimatch from 'minimatch';
 import { collect } from 'collect.js';
 import twigPreset from '../twig.js';
 import markdown from '../markdown.js';
